pkgs/os-specific/linux/wpa_supplicant/default.nix

```diff
@@ -66,6 +66,7 @@ stdenv.mkDerivation rec {
     CONFIG_WPS=y
     CONFIG_WPS_ER=y
     CONFIG_WPS_NFS=y
+    CONFIG_MACSEC=y
```

Comment:
This is already enabled by default:
https://w1.fi/cgit/hostap/tree/wpa_supplicant/defconfig?h=hostap_2_10

Reply:
Yep, must have overlooked that while testing.